src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -15,7 +15,20 @@
  */
 public class Intake extends SubsystemBase {
 
-    public static final double INTAKE_RATE = 0.01;
+    /**
+     * Enum for Intake States.
+     */
+    public enum States {
+        UP,
+        DOWN,
+        UNKNOWN
+    }
+
+    public States state;
+
+    public static final double
+            INTAKE_RATE = 0.01,
```

Comment:
the rate is unnessary, you arnt useing it anywhere

Reply:
It actually is in use for defining the rate limiter.

Reply:
beacuse we never have to use pid on the intake, it would be better to use openRampRate witch is built into the spark to limit acceloration, as that will reduce the number of can messages sent